plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -51,183 +100,2143 @@ def get_ilo_sessions(self):
         result["ret"] = True
         return result
 
+    def get_systems_data(self):
+        response = self.get_request(self.root_uri + self.systems_uri)
+        if not response["ret"]:
+            return response
+
+        return {
+            "ret": True,
+            "msg": response
+        }
+
+    def get_managers_data(self):
```

Comment:
Why is this iLO specific? Getting manager information is standard and covered by the base module.